cmd/podman/commit.go

```diff
@@ -120,7 +121,17 @@ func commitCmd(c *cli.Context) error {
 		Changes:       c.StringSlice("change"),
 		Author:        c.String("author"),
 	}
-	newImage, err := ctr.Commit(getContext(), reference, options)
+	var createArtifact createConfig
+	artifact, err := ctr.GetArtifact("create-config")
```

Comment:
If error is not nil on this, maybe a logrus.Info saying we can't retrieve information on volumes, command, entrypoint?

Reply:
i prefer the hard fail... the user expects this to work, continuing without doing it should be a hard fail.

Reply:
Right now, we only unmarshal the artifact if it exists - if `err` is not nil on this line, we do nothing. This could happen with CRI-O containers, where we don't have enough info saved right now to reliably commit.

Maybe we should start storing all of this in the DB after all... I have a PR open to add some related things to the DB already, maybe it should be extended to also add the full volumes, entrypoint, command from when the container was created, so we can reliably handle things like CRI-O containers?